wiki/People/The_Team/Global_Moderation_Team/de.md

```diff
@@ -61,10 +61,10 @@ Die [Seite für das globale Moderationsteam](https://osu.ppy.sh/groups/4) listet
 | ![][flag_HU] [[ Another ]](https://osu.ppy.sh/users/3416573) | Ungarisch | Chat-Moderation |
 | ![][flag_CH] [\[ryuu\]](https://osu.ppy.sh/users/5698467) | Russisch | Chat-Moderation |
 | ![][flag_US] [abraker](https://osu.ppy.sh/users/4635891) |  | Forum-Moderation |
-| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) |  | Verwaltung von Turnieren |
+| ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578) |  | Verwaltung von Wettbewerben |
```

Comment:
I just noticed that the `group-info` file in the meta folder has an incorrect value. `Turnier` and `Wettbewerb` have a large difference in usage on osu! related topics, as can be [seen in the German ASC](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing#%C3%BCbersetzungen-f%C3%BCr-h%C3%A4ufige-begriffe). In this section are common terms and their correct translation listed. `Turnier` is used only ***and only*** for tournaments. In the contrary, `Wettbewerb` is used only for contests.

I would very much appreciate it if the changed files are fixed in this PR. Otherwise, I can do it in a separate one. I will open a PR for the `group-info` file later or tomorrow.

Reply:
i've fixed this for GMT here in [ebaac52](https://github.com/ppy/osu-wiki/pull/7189/commits/ebaac52d6e16df6f95747635444d0581fc89c190), but do pr the change to `group-info`

Reply:
Thanks, will do 👍